assets/cubyz/biomes/cave/void/void_roots.zig.zon

```diff
@@ -12,11 +12,18 @@
 	.structures = .{
 		.{
 			.id = "cubyz:simple_vegetation",
-			.chance = 0.001,
+			.chance = 0.00001,
```

Comment:
I think you added a zero too much here. I flew around for ~30000 blocks and only encountered 2 torches.